src/main/java/creativei/service/impl/ReminderServiceImpl.java

```diff
@@ -1,40 +1,65 @@
 package creativei.service.impl;
 
 import creativei.dao.ReminderCustomDao;
-import creativei.dao.ReminderDao;
 import creativei.entity.Reminder;
 import creativei.exception.InvalidParamRequest;
 import creativei.service.ReminderService;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
+import util.LocalizationUtil;
 import util.StringUtil;
 
+import java.text.ParseException;
+import java.util.Date;
+import java.util.List;
+
 @Service("ReminderService")
-public class ReminderServiceImpl implements ReminderService{
+public class ReminderServiceImpl implements ReminderService {
     @Autowired
     ReminderCustomDao reminderCustomDao;
 
-    Logger logger= LoggerFactory.getLogger(ReminderServiceImpl.class);
+    Logger logger = LoggerFactory.getLogger(ReminderServiceImpl.class);
 
     @Override
     public Reminder create(Reminder reminder) throws InvalidParamRequest {
         logger.info(" Service reminder create");
-        if(!validateReminder(reminder)){
+        if (!validateReminder(reminder)) {
             throw new InvalidParamRequest("Required Field are null");
         }
         return reminderCustomDao.save(reminder);
     }
 
-    private Boolean validateReminder(Reminder reminder){
+    @Override
+    public List<Reminder> getReminderByDateRange(String param1, String param2) throws ParseException, InvalidParamRequest {
+        logger.info("getReminderDateRange Method");
+        Date toDate=LocalizationUtil.stringtoDateWithTimeConverter(param1);
+        Date fromDate=LocalizationUtil.stringtoDateWithTimeConverter(param2);
+
+        if(toDate.after(fromDate))
```

Comment:
Invalid error message. Should be "invalid date range" and use following for logging- "invalid date range- toDate is smaller than fromDate"

Reply:
logger error message changed to suggested one 3fe67e5